core/src/main/java/io/ddf/util/Utils.java

```diff
@@ -66,6 +69,46 @@ public static DataFormat getDataFormatFromPath(String path) throws DDFException
     }
   }
 
+
+  public static List<String> sanitizeColumnName(List<String> columnNames) throws DDFException {
+    Preconditions.checkArgument(columnNames.size() > 0);
+    List<String> ret = new ArrayList<>();
+    for (String name : columnNames) {
+      Preconditions.checkArgument(name.length() > 0);
+      String dashName = name.trim().replaceAll("![0-9a-zA-Z]|\\s", "_");
+      int i;
+      for (i = 0; i < dashName.length(); ++i) {
```

Comment:
@nhanitvn @hai-adatao This is to replace space and non-valid characters such as "@" in the column names